i18n/extra/es.json

```diff
@@ -93,8 +93,8 @@
             "SMW_YM"
         ],
         [
-            "SMW_MDY",
             "SMW_DMY",
+            "SMW_MDY",
```

Comment:
This change influences the preference [0] in how the Time/Date is interpret from a string [1] therefore I'd like to ask users with ES language competence (@jaideraf) to comment on this to avoid introducing a sub-optimal change in response to [2].

[0] https://github.com/SemanticMediaWiki/SemanticMediaWiki/blob/master/src/ExtraneousLanguage/ExtraneousLanguage.php#L427-L430
[1] https://github.com/SemanticMediaWiki/SemanticMediaWiki/blob/eff53c2ff1fd0365c823c4ae867aed57749edcf2/includes/datavalues/SMW_DV_Time.php#L461-L473
[2] https://www.mediawiki.org/w/index.php?title=Extension_talk:Semantic_Result_Formats/calendar_format&curid=40988&diff=2288909&oldid=2287003